services/scan/src/scanners/fujitsu.ts

```diff
@@ -157,9 +200,14 @@ export class FujitsuScanner implements Scanner {
       endBatch: async (): Promise<void> => {
         if (!done) {
           done = true;
-          debug(
-            'scanimage [pid=%d] stopping scan by closing stdin',
-            scanimage.pid
+          void this.logger.log(
```

Comment:
I can't comment on the actual line but ... @eventualbuddha curious if you think I should wrap the debug on line 178 in a call to logger.log, I didn't really understand what was happening there well enough to know what the log should be so I just left it for now.

Reply:
I don't think that needs to be in the user-facing logs. It's just saying that it's sending `\n\n` to the stdin of the `scanimage` process to trigger the scanning of two pages (one sheet), which works because `scanimage` was spawned with the `--batch-prompt` option.